packages/grafana-ui/src/components/Dropdown/ButtonSelect.tsx

```diff
@@ -86,7 +86,7 @@ const getStyles = (theme: GrafanaTheme) => {
     menuWrapper: css`
       position: absolute;
       z-index: ${theme.zIndex.dropdown};
-      top: ${theme.spacing.formButtonHeight + 1}px;
+      top: ${theme.spacing(4)};
```

Comment:
Does it make sense to make this one digit smaller than the initial value? Used `32px` instead of `33px`.
@torkelo